src/v2/guide/forms.md

```diff
@@ -342,42 +338,42 @@ vm.pick === vm.a
 ```
 
 ``` js
-// when selected:
+// khi được chọn:
 typeof vm.selected // => 'object'
 vm.selected.number // => 123
 ```
 
-## Modifiers
+## Modifier
 
 ### `.lazy`
 
-By default, `v-model` syncs the input with the data after each `input` event (with the exception of IME composition as [stated above](#vmodel-ime-tip)). You can add the `lazy` modifier to instead sync after `change` events:
+Mặc định, `v-model` đồng bộ giá trị của input với dữ liệu sau sự kiện `input` (có ngoại lệ đối với trường hợp bộ gõ IME như [đã nói ở trên](#vmodel-ime-tip)). Bạn có thể thêm modifier `lazy` để đồng bộ sau sự kiện `change`:
 
 ``` html
-<!-- synced after "change" instead of "input" -->
+<!-- được đồng bộ sau sự kiện "change" thay vì "input" -->
 <input v-model.lazy="msg" >
 ```
 
 ### `.number`
 
-If you want user input to be automatically typecast as a number, you can add the `number` modifier to your `v-model` managed inputs:
+Nếu bạn muốn giá trị từ input được tự động chuyển kiểu về kiểu số, bạn có thể thêm modifier `number` vào `v-model`:
 
 ``` html
 <input v-model.number="age" type="number">
 ```
 
-This is often useful, because even with `type="number"`, the value of HTML input elements always returns a string.
+Điều này rất hữu ích, vì kể cả khi input có `type="number"` thì giá trị được trả về của nó luôn luôn là chuỗi.
 
 ### `.trim`
 
-If you want user input to be trimmed automatically, you can add the `trim` modifier to your `v-model` managed inputs:
+Dùng để tự động cắt tỉa giá trị trong input.
 
 ```html
 <input v-model.trim="msg">
 ```
 
-## `v-model` with Components
+## `v-model` với Component
 
-> If you're not yet familiar with Vue's components, you can skip this for now.
+> Nếu bạn chưa quen với Vue component, bạn có thể bỏ qua nó.
 
-HTML's built-in input types won't always meet your needs. Fortunately, Vue components allow you to build reusable inputs with completely customized behavior. These inputs even work with `v-model`! To learn more, read about [custom inputs](components.html#Form-Input-Components-using-Custom-Events) in the Components guide.
+Không phải lúc nào các kiểu input có sẵn của HTML cũng có thể đáp ứng nhu cầu của bạn. Thật may là Vue component cho phép bạn xây dựng các input riêng với các hành vi có khả năng tùy chỉnh hoàn toàn và khả năng sử dụng lại. Nó thậm chí còn hoạt động với `v-model`! Để xem thêm, hãy đọc [input tùy chỉnh](components.html#Form-Input-Components-using-Custom-Events).
```

Comment:
"Thật may là Vue component cho phép bạn xây dựng các input riêng với các hành vi có khả năng tùy chỉnh hoàn toàn và khả năng sử dụng lại."

Đoạn này anh sẽ dịch thành:

"May mắn là các component của Vue cho phép bạn xây dựng các input riêng tái sử dụng được với các hành vi (behavior) được tùy chỉnh hoàn toàn. Các input này thậm chí còn hoạt động với `v-model`."